test/fixtures/cloudbuild_enabled/versions.tf

```diff
@@ -15,5 +15,5 @@
  */
 
 terraform {
-  required_version = ">=0.12.20"
```

Comment:
nit: unrelated but we can actually remove this from fixtures since its just for our usage.

Reply:
SG - I'll remove the entire file as this is the only item in them.